src/libraries/Common/tests/Tests/System/IO/StreamConformanceTests.cs

```diff
@@ -905,7 +905,7 @@ public virtual async Task Read_PopulatedWithInitialData_ToEof_Success(ReadWriteM
                 Assert.Equal(size, stream.Seek(0, SeekOrigin.Current));
             }
 
-            Assert.Equal(expected, actual.ToArray());
+            Assert.True(expected.AsSpan().SequenceEqual(actual.ToArray()));
```

Comment:
```suggestion
            AssertExtensions.Equal(expected, actual.ToArray());
```

Reply:
I'm happy to make this change, since it seems like `AssertExtensions.Equal` is the correct method to use.

That said, I really wish this method was named differently, e.g. `AssertExtensions.SequenceEquals` or something. As it stands, it's not obvious from looking at the code that it's actually doing the right thing here (unlike `Assert.Equals`). And I never would have found this on my own without you pointing me to it -- I actually did look on AssertExtensions, but was expecting something like SequenceEquals or SpanEquals or whatever, not just Equal.